common-content/en/module/decomposition/deploy-trivial-frontend-and-backend/index.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Deploy a frontend and backend"
+headless = true
+time = 60
+facilitation = false
+emoji= "📖"
+objectives = [
+  "Deploy a frontend and backend",
+  "Configure a frontend to talk to a specific deployed backend",
+]
++++
+
+Websites tend to only be useful if they're running somewhere that people can access them.
+
+We need to deploy our frontend and our backend on the internet so that people can use them.
+
+And they need to know how to talk to each other. In your frontend you probably hard-coded your `fetch` to fetch from `http://127.0.0.1:3000`.
+
+First let's deploy our backend somewhere. Then when we know its address, we can update our frontend to talk to our deployed backend, and then deploy it too.
```

Comment:
I think I want some pictures. Or some diagrams?

Reply:
Added a TODO for now - will fill this in when we have a backend hosting platform confirmed.